types/react-dom/index.d.ts

```diff
@@ -1,63 +1,65 @@
-// Type definitions for React (react-dom) 0.14
+// Type definitions for React (react-dom) 15.5
 // Project: http://facebook.github.io/react/
-// Definitions by: Asana <https://asana.com>, AssureSign <http://www.assuresign.com>, Microsoft <https://microsoft.com>
+// Definitions by: Asana <https://asana.com>, AssureSign <http://www.assuresign.com>, Microsoft <https://microsoft.com>, MartynasZilinskas <https://github.com/MartynasZilinskas>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.1
 
-export as namespace ReactDOM;
```

Comment:
This line was removed -- is the package no longer available as a global variable `ReactDOM`?